shared/models/plugins/server-hook.model.ts

```diff
@@ -9,9 +9,13 @@ export const serverFilterHookObject = {
   // Used to get detailed video information (video watch page for example)
   'filter:api.video.get.result': true,
 
-  // Filter the result of the accept upload function
+  // Filter the result of the accept upload, import via torrent or url functions
```

Comment:
It means `filter:api.video.upload.accept.result` will also hook an import after downloading the video inside a job, how to inform the client ?

Reply:
No, it will only work for the direct upload.

Reply:
You cannot inform the client. I should check if a notification is sent.

Reply:
Maybe, we should inform with persistent message until the user delete the inserted import infos.

Via a simple alert in the video-watch view with a message like "Video import has been rejected."
[![Screenshot-import-rejected.png](https://i.postimg.cc/yxmbRZQr/Screenshot-import-rejected.png)](https://postimg.cc/ctJhyrNQ)

And a status message in the video-list view: 

[![Screenshot-import-rejected-list-png.png](https://i.postimg.cc/gJQdr6SZ/Screenshot-import-rejected-list-png.png)](https://postimg.cc/CBG9QdVh)

if it's too much work, I can submit another PR ?

Reply:
WDYT @Chocobozzz ?

Reply:
Not a bad idea: I think we can add another entry in the `VideoState` enum

Reply:
@Chocobozzz @rigelk sorry to come back on this PR but the client message should be implemented in another PR ?

Reply:
The `errorMessage` of your hook is displayed in the video import table

Reply:
Thanks @Chocobozzz, it sounds logic.